repository/c.json

```diff
@@ -3544,6 +3544,16 @@
 				}
 			]
 		},
+		{
+			"name": "Continuous Merge",
+			"details": "https://github.com/vim-zz/sublime-cm-syntax",
+			"releases": [
```

Comment:
Final note: to help users find your package, you might want to add the "language syntax" label.